brainstormforce/astra
# AST-2336 - Footer builder area breaking in customizer due to columns control present in their row layouts

### Description
- Problem arises because of footer row column control
- More details - https://brainstormforce.atlassian.net/browse/AST-2336

### Screenshots
- https://d.pr/v/RZ7xUV

### Types of changes
- Bug fix

### How has this been tested?
- Make sure you are running with WP 6.1
- Go to Footer Builder and try editing the Above or Primary or Below footer

### Checklist:
